sdk/python/kfp/compiler/compiler.py

```diff
@@ -143,6 +143,10 @@ def _op_to_template(self, op):
       template['container']['env'] = list(map(self._convert_k8s_obj_to_dic, op.env_variables))
     if op.volume_mounts:
       template['container']['volumeMounts'] = list(map(self._convert_k8s_obj_to_dic, op.volume_mounts))
+
+    if op.pod_annotations:
```

Comment:
Can you also add labels so that the metadata support is complete?

Reply:
done